packages/@react-stately/calendar/src/useCalendarState.ts

```diff
@@ -33,6 +33,8 @@ export function useCalendarState(props: CalendarProps): CalendarState {
 
     if (!isSameMonth(date, currentMonth)) {
       setCurrentMonth(startOfMonth(date));
+      requestAnimationFrame(() => setFocusedDate(date));
```

Comment:
I don't think we want requestAnimationFrame in the stately package, that's browser specific, please move it out to the react-spectrum package somehow

Reply:
So should this be in the useUpdateEffect: https://github.com/adobe-private/react-spectrum-v3/pull/188/files/86c5dad726c55b0dd5dcbdb185368a98851459fc#diff-aa122aaed571a90e88b8a601bcbf7afcR73-R77

Reply:
umm... not sure where it should go because i haven't wrapped my head around the rest, i just know it shouldn't be in stately :)

Reply:
I think I can remove the `requestAnimationFrame`, because the method in `useCalendarBase` that sets focus to the appropriate cell after a focused date change, `focusFocusedDateCell`, is wrapped in a `useCallback` method.